From e3b7afbff2b2b34ec4092fcc1dd3211cbdcf869b Mon Sep 17 00:00:00 2001
From: Jiri Olsa <jolsa@redhat.com>
Date: Mon, 25 Aug 2014 07:51:14 -0400
Subject: [tools] perf: Making QUIET_(CLEAN

Message-id: <1408953210-26343-187-git-send-email-jolsa@redhat.com>
Patchwork-id: 88972
O-Subject: [PATCH RHEL7.1 BZ1133083 186/322] perf tools: Making QUIET_(CLEAN|INSTAL) variables global
Bugzilla: 1133083
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Jiri Olsa <jolsa@kernel.org>

Bugzilla: 1133083
https://bugzilla.redhat.com/show_bug.cgi?id=1133083

upstream
========
commit b7248defce917acce74b338d69c2b8f600370213
Author: Jiri Olsa <jolsa@redhat.com>
Date: Thu Dec 19 14:42:00 2013 +0100

description
===========
Moving QUIET_(CLEAN|INSTAL) variables into:

  tools/scripts/Makefile.include

to be usable by other tools. The change to use them in libtraceevent is
in following patches.
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/tools/perf/config/utilities.mak b/tools/perf/config/utilities.mak
index f168deb..4d985e0 100644
--- a/tools/perf/config/utilities.mak
+++ b/tools/perf/config/utilities.mak
@@ -178,10 +178,3 @@ endef
 _ge_attempt = $(if $(get-executable),$(get-executable),$(_gea_warn)$(call _gea_err,$(2)))
 _gea_warn = $(warning The path '$(1)' is not executable.)
 _gea_err  = $(if $(1),$(error Please set '$(1)' appropriately))
-
-ifneq ($(findstring $(MAKEFLAGS),s),s)
-  ifneq ($(V),1)
-    QUIET_CLEAN  = @printf '  CLEAN    %s\n' $1;
-    QUIET_INSTALL = @printf '  INSTALL  %s\n' $1;
-  endif
-endif
diff --git a/tools/scripts/Makefile.include b/tools/scripts/Makefile.include
index ee76544..e4cfbed 100644
--- a/tools/scripts/Makefile.include
+++ b/tools/scripts/Makefile.include
@@ -76,5 +76,8 @@ ifneq ($(findstring $(MAKEFLAGS),s),s)
   +@echo        '  DESCEND  '$(1); \
   mkdir -p $(OUTPUT)$(1) && \
   $(MAKE) $(COMMAND_O) subdir=$(if $(subdir),$(subdir)/$(1),$(1)) $(PRINT_DIR) -C $(1) $(2)
+
+ QUIET_CLEAN    = @printf '  CLEAN    %s\n' $1;
+ QUIET_INSTALL  = @printf '  INSTALL  %s\n' $1;
   endif
 endif
-- 
1.7.1